FAO Publications.js

```diff
@@ -664,40 +657,122 @@ var testCases = [
 				],
 				"tags": [
 					{
-						"tag": "Asia and the Pacific"
+						"tag": "meetings"
 					},
 					{
-						"tag": "FAO"
+						"tag": "traditional foods"
+					},
+					{
+						"tag": "亚太"
+					},
+					{
+						"tag": "人体营养"
+					},
+					{
+						"tag": "粮食供给"
 					},
 					{
-						"tag": "Sustainable Development Goals"
+						"tag": "联合国粮农组织"
 					},
 					{
-						"tag": "agricultural development"
+						"tag": "营养不良"
 					},
 					{
-						"tag": "climate change"
+						"tag": "营养政策"
 					},
 					{
-						"tag": "development indicators"
+						"tag": "营养教育"
 					},
 					{
-						"tag": "development policies"
+						"tag": "营养状况调查"
 					},
 					{
-						"tag": "food safety"
+						"tag": "营养监督"
 					},
 					{
-						"tag": "food security"
+						"tag": "进食"
 					},
 					{
-						"tag": "human nutrition"
+						"tag": "食品供应链"
+					},
+					{
+						"tag": "食品工业"
+					},
+					{
+						"tag": "食物政策"
+					},
+					{
+						"tag": "食物的获得"
+					},
+					{
+						"tag": "食物链"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.fao.org/publications/card/en/c/5014f143-be17-4b58-b90e-f1c6bef344a0/",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Climate-Smart Agriculture: A Call for Action: Synthesis of the Asia-Pacific Regional Workshop Bangkok, Thailand, 18 to 20 June 2015",
+				"creators": [
+					{
+						"lastName": "FAO",
+						"creatorType": "author",
+						"fieldMode": 1
+					}
+				],
+				"date": "2015",
+				"ISBN": "9789251088630",
+				"abstractNote": "This publication is a summary of the workshop held in Bangkok, Thailand from 18 to 20 June 2015 to promote the mainstreaming and up-scaling of Climate-Smart Agriculture in the region. Included in the report are successful case studies that agriculturists have been practicing as a means to address food security under adverse circumstances.",
+				"language": "other",
```

Comment:
I pushed the PR to get it out to users, but we should fix this. I didn't really look into it, but the URL itself has 'en' in it, and the PDF itself has 'E' in it (which may mean English?), so it seems like we should be able to avoid `other` here.